src/components/NetworkMap.vue

```diff
@@ -134,22 +134,47 @@
         width="4"
         height="4"
         transform="translate(-2,-2)"
+        v-if="!emoji || !egoEmoji"
       />
+      <text
+        v-if="emoji && egoEmoji"
+        :x="egoCoords[0]"
+        :y="egoCoords[1]"
+        class="mark"
+        style="font-size: 4px; cursor: pointer; text-anchor: middle"
+      >
```

Comment:
in meinem Firefox und Chrome ist das Emoji nicht vertikal zentriert (v.a. bei Ego deutlich sichtbar).
Vmt. wird es als Text so positioniert, dass bei :y die Baseline ist. 
Mit style dy: -1px ist es evt. näher drann.